src/System.Security.Cryptography.Algorithms/tests/AesCcmTests.cs

```diff
@@ -411,6 +498,9 @@ public static IEnumerable<object[]> GetNistCcmTestCases()
                 CaseId = 0,
                 Key = "eda32f751456e33195f1f499cf2dc7c97ea127b6d488f211ccc5126fbb24afa6".HexToByteArray(),
                 Nonce = "a544218dadd3c1".HexToByteArray(),
+                Plaintext = Array.Empty<byte>(),
+                AssociatedData = null,
```

Comment:
Is there a semantic difference between null and empty for AssociatedData?

Reply:
@bartonjs - no - both mean zero length